test/test_init.py

```diff
@@ -10,6 +10,9 @@
 logger = logging.getLogger(__name__)
 
 
+TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
Ik ben trouwens (ook in threeditoolbox, kijk maar in `dependencies.py`) op python 3's pathlib aan het overstappen.
Als je er hier een `Path` van maakt kan je verderop de pathlib geintjes als `TEST_DATA_DIR / "mijn.sql"` gebruiken.

Het was even wennen, maar ik vind die pathlib nu wel erg lekker.

Reply:
done